bigip/resource_bigip_ltm_pool.go

```diff
@@ -187,13 +186,15 @@ func resourceBigipLtmPoolUpdate(d *schema.ResourceData, meta interface{}) error
 		AllowNAT:          d.Get("allow_nat").(string),
 		AllowSNAT:         d.Get("allow_snat").(string),
 		LoadBalancingMode: d.Get("load_balancing_mode").(string),
-		Description:       d.Get("description").(string),
 		SlowRampTime:      d.Get("slow_ramp_time").(int),
 		ServiceDownAction: d.Get("service_down_action").(string),
 		ReselectTries:     d.Get("reselect_tries").(int),
 		Monitor:           strings.Join(monitors, " and "),
 	}
 
+	if d.Get("description").(string) != "" {
```

Comment:
I don't think you need this conditional, if the bigip returns an empty string, that's what we should store in the state.

Reply:
@dannyk81 
Updated with comments and submitted change